phoenix-core/src/main/java/org/apache/phoenix/iterate/BaseResultIterators.java

```diff
@@ -39,6 +39,7 @@
 import java.io.EOFException;
 import java.sql.SQLException;
 import java.util.*;
+import java.util.Arrays;
```

Comment:
Can you remove this import please or list everything needed under java.util?

Reply:
You are right, but this has already been committed.
Can you open a follow-up ticket ?